packages/wrangler/src/__tests__/api/startDevWorker/LocalRuntimeController.test.ts

```diff
@@ -1047,9 +1047,21 @@ describe("LocalRuntimeController", () => {
 			expect(res.status).toBe(204);
 			expect(await reportPromise).toEqual(["message"]);
 		});
-		it("should expose hyperdrive bindings", async () => {
-			// Start echo TCP server
-			const server = net.createServer((socket) => socket.pipe(socket));
+		it("should expose hyperdrive bindings - default", async () => {
+			// Start TCP echo server
+			const POSTGRES_SSL_REQUEST_PACKET = Buffer.from([
+				0x00, 0x00, 0x00, 0x08, 0x04, 0xd2, 0x16, 0x2f,
+			]);
+			const server = net.createServer((socket) => {
+				socket.on("data", (chunk) => {
+					if (POSTGRES_SSL_REQUEST_PACKET.equals(chunk)) {
+						socket.write("N");
+					} else {
+						// echo back result
+						socket.write(chunk);
+					}
+				});
```

Comment:
I feel like I have been seeing this code many times at this point in the review.

Is there an opportunity to factor some code ?